### ISSUE TITLE
Problem when exporting Payment Tokens

### ISSUE THREAD

**Issue by alarocca130** [woocommerce/woocommerce#20100]
**tl;dr**
When exporting Payment Tokens, exporter method (\WC_Privacy_Exporters::customer_tokens_exporter) returns a malformed data structure to WordPress, causing the following PHP warning

PHP Warning:  array_merge(): Argument 1 is not an array in /srv/www/local/htdocs/wp-admin/includes/file.php on line 2215

**Bug Description**
WordPress requires from each exporter the following data structure:
```
[
	data: [
		[
			group_id: (string),
			group_label: (string),
			item_id: (string),
			data: [
				[
					name: (string),
					value: (string)
				],
				...
			]
		],
		...
	],
	done: (bool)
]
```
Anyway, instead of returning a structure with name and value, Payment Token exporter just returns a string describing Payment Method

`'data'        => $token->get_display_name(),`

When WordPress use these data to format export items, performs an `array_merge`; since first parameter is not an array, a warning will be thrown

`$merged_item_data = array_merge( $export_datum['data'], $old_item_data );`

In second place, parameter `done` should be used to control pagination process, and stop it once there are no more items to export

Anyway, even if exporter correctly handles paginated items, it will always return done as true, exporting just first page of tokens

`'done' => true,`

**To Reproduce**
1. Make sure to register some Payment Token with any Payment Gateway that supports tokenization
2. Create a Personal Data Export, in WP Dashboard -> Tools -> Export Personal Data
3. Download Personal Data export

**Screenshots**
Export file: https://www.screencast.com/t/d7On5smyjdn
Debug log: https://www.screencast.com/t/XVPU6v5T

**Expected behavior**
Exporter method should return correct data structure, to let WordPress generate HTML export for Payment Tokens
Besides, exporter should allow admin export more than 10 tokens (1st page)

**Isolating the problem:**
- [ ] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [x] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [x] I can reproduce this bug consistently using the steps above.

**WordPress Environment**
<details>
### WordPress Environment ###

Home URL: http://local.test/stripe
Site URL: http://local.test/stripe
WC Version: 3.4.0
Log Directory Writable: ✔
WP Version: 4.9.6-beta1-43171
WP Multisite: ✔
WP Memory Limit: 256 MB
WP Debug Mode: ✔
WP Cron: ✔
Language: en_US
External object cache: –

### Server Environment ###

Server Info: nginx/1.13.9
PHP Version: ❌ 7.0.28-1+ubuntu14.04.1+deb.sury.org+1 - We recommend using PHP version 7.2 or above for greater performance and security. How to update your PHP version
PHP Post Max Size: 1 GB
PHP Time Limit: 30
PHP Max Input Vars: 1000
cURL Version: 7.35.0
OpenSSL/1.0.1f

SUHOSIN Installed: –
MySQL Version: ❌ 5.5.5 - We recommend a minimum MySQL version of 5.6. See: WordPress requirements
Max Upload Size: 1 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 3.3.5
WC Database Prefix: wp_2_
MaxMind GeoIP Database: ✔
Total Database Size: 29.38MB
Database Data Size: 18.36MB
Database Index Size: 11.02MB
wp_2_woocommerce_sessions: Data: 0.02MB + Index: 0.02MB
wp_2_woocommerce_api_keys: Data: 0.02MB + Index: 0.03MB
wp_2_woocommerce_attribute_taxonomies: Data: 0.02MB + Index: 0.02MB
wp_2_woocommerce_downloadable_product_permissions: Data: 0.02MB + Index: 0.05MB
wp_2_woocommerce_order_items: Data: 0.02MB + Index: 0.02MB
wp_2_woocommerce_order_itemmeta: Data: 0.14MB + Index: 0.09MB
wp_2_woocommerce_tax_rates: Data: 0.02MB + Index: 0.06MB
wp_2_woocommerce_tax_rate_locations: Data: 0.02MB + Index: 0.03MB
wp_2_woocommerce_shipping_zones: Data: 0.02MB + Index: 0.00MB
wp_2_woocommerce_shipping_zone_locations: Data: 0.02MB + Index: 0.03MB
wp_2_woocommerce_shipping_zone_methods: Data: 0.02MB + Index: 0.00MB
wp_2_woocommerce_payment_tokens: Data: 0.02MB + Index: 0.02MB
wp_2_woocommerce_payment_tokenmeta: Data: 0.02MB + Index: 0.03MB
wp_2_woocommerce_log: Data: 0.02MB + Index: 0.02MB
wp_2_commentmeta: Data: 0.02MB + Index: 0.03MB
wp_2_comments: Data: 0.09MB + Index: 0.09MB
wp_2_links: Data: 0.02MB + Index: 0.02MB
wp_2_options: Data: 1.06MB + Index: 0.02MB
wp_2_postmeta: Data: 0.48MB + Index: 0.58MB
wp_2_posts: Data: 0.08MB + Index: 0.06MB
wp_2_termmeta: Data: 0.02MB + Index: 0.03MB
wp_2_terms: Data: 0.02MB + Index: 0.03MB
wp_2_term_relationships: Data: 0.02MB + Index: 0.02MB
wp_2_term_taxonomy: Data: 0.02MB + Index: 0.03MB
wp_2_wc_download_log: Data: 0.02MB + Index: 0.03MB
wp_2_wc_webhooks: Data: 0.02MB + Index: 0.02MB
wp_2_yith_wc_stripe_blacklist: Data: 0.02MB + Index: 0.05MB
wp_2_yith_ywsbs_activities_log: Data: 0.05MB + Index: 0.00MB

### Post Type Counts ###

nav_menu_item: 4
page: 5
post: 2
product: 21
revision: 1
shop_coupon: 1
shop_order: 81
shop_order_refund: 1
user_request: 1
ywsbs_subscription: 56

### Security ###

Secure connection (HTTPS): ❌
					Your store is not using HTTPS. Learn more about HTTPS and SSL Certificates.
Hide errors from visitors: ❌Error messages should not be shown to visitors.

### Active Plugins (1) ###

WooCommerce: by Automattic – 3.4.0-rc.1

### Settings ###

API Enabled: –
Force SSL: –
Currency: USD ($)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
variable (variable)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

### WC Pages ###

Shop base: 4 - /
Cart: 5 - /cart/
Checkout: 6 - /checkout/
My account: 7 - /my-account/
Terms and conditions: ❌ Page not set

### Theme ###

Name: Twenty Seventeen
Version: 1.5
Author URL: https://wordpress.org/
Child Theme: ❌ – If you are modifying WooCommerce on a parent theme that you did not build personally we recommend using a child theme. See: How to create a child theme
WooCommerce Support: ✔

### Templates ###

Overrides: –
</details>

**Comment by mikejolley** [comment]
See https://github.com/woocommerce/woocommerce/pull/20131
